utility/getdata_setup.xml

```diff
@@ -3,12 +3,15 @@
     <title>getdata_setup</title>
   </head>
   <body>
-    <dummy_directory>dummy</dummy_directory>
-    <common_local_data_directory>common_data</common_local_data_directory><!--requires relative path-->
-    <user_local_data_directory>data</user_local_data_directory><!--requires relative path-->
-    <server_private_address>data@deep.reak.bme.hu:~/private_html/renate-od</server_private_address>
-    <private_key>data/deep-data.ppk</private_key>
+    <server_address>deep.reak.bme.hu</server_address>
+    <server_private_data>private_html/renate-od</server_private_data>
+    <server_public_data>public_html/renate-od</server_public_data>
+    <user_name>data</user_name>
     <server_public_address>http://deep.reak.bme.hu/~data/renate-od</server_public_address>
+    <private_key>data/deep_data_key</private_key>
```

Comment:
New keys to be generated.

Reply:
New key generated for @asztalosors  and @andorfi . Other user access remianed unaffected, as everyone has their own key.